tortoise/ninja_tortoise.go

```diff
@@ -16,8 +16,9 @@ type vec [2]int
 type patternID uint32 // this hash does not include the layer id
 
 const ( // Threshold
-	window          = 10
-	globalThreshold = 0.6
+	window            = 10
+	globalThreshold   = 0.5
+	newLayerThreshold = 0.5 * (globalThreshold / 0.6) // adjust to match global threshold
```

Comment:
I'd explain better why this adjustment is made and why like so (the result of this as I understand is 0.41 and I have no idea why this is the correct value)